book.rb

```diff
@@ -0,0 +1,14 @@
+class Book
+  attr_accessor :title, :author
+  attr_reader :rentals
```

Comment:
- [ ] This `rentals` variable should have a getter and setter, instead of only getter. Kindly fix that

Reply:
Please according to this diagram the `rentals` attr is private (read-only) and should not be writable. @julie-ify 

![image](https://user-images.githubusercontent.com/81693905/174063101-155f7474-1f91-4fcd-a899-484a3c0d041b.png)